src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -222,13 +222,12 @@ function MultiCombobox<Item>({
             onMouseDown: toggleMenu,
             readOnly: true,
             placeholder: !value.length ? placeholder : '',
-            position: !value.length ? 'static' : 'absolute',
           }),
           ...(searchable && {
-            placeholder,
-            mt: (isOpen && value.length) || (isOpen && value.length && allowAdditions) ? -4 : 0,
-            position: isOpen || (isOpen && value.length && allowAdditions) ? 'static' : 'absolute',
+            placeholder: hideLabel && value.length && !isOpen ? '' : placeholder,
+            p: isOpen ? 1 : null,
           }),
+          position: 'absolute',
```

Comment:
Changed position to absolute:
- When the dropdown is closed the input will cover the entire dropdown.
- When the dropdown is opened it will be inlined in the `Tag` list and have the same width and height as the hidden span.

Reply:
Is there a way to have the input be present in the same line in some edge cases?

<img width="436" alt="Screen Shot 2021-03-02 at 12 51 04 PM" src="https://user-images.githubusercontent.com/10436045/109638107-46ca5080-7b56-11eb-82ce-127211034dd2.png">
<img width="404" alt="Screen Shot 2021-03-02 at 12 51 06 PM" src="https://user-images.githubusercontent.com/10436045/109638112-47fb7d80-7b56-11eb-9088-a475577c424a.png">

I would have expected the following:

- The input shows in the same line for the first 1-2 characters
- It moves in its own line when it doesn't fit anymore

Reply:
This is because i added `minWidth="60px"` to the input. I'll remove it.